### ISSUE TITLE
param "user_max_follow" and "user_min_follow" doesn't work

### ISSUE THREAD

**Issue by wdstk** [instabot-py/instabot.py#2373]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [x] Are you running the code on your phone 
- [ ] If so, have you tried running the code on your local machine?

#### Operating System? (include version)
- [ ] macOS
- [ ] Windows
- [ ] Rasbian Linux Raspberry PI (include flavour)
- [ ] Linux (include flavour)
- [x] Android (include terminal)

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
python 3.7.4
```

#### Pip Version?
```
pip 19.2.2
```

#### Instabot Version?
```
instabot-py 0.5.9
```

#### How you use the instabot
- [x] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue

If I set params "user_max_follow" and "user_min_follow", the bot never do follow with message "Not following {username}, the account doesn't meet requirements" although the number of followers is in the range.

I think verify_account_followers() in instabot.py doesn't return True in case the params were set and the number is in the range.

```
    def verify_account_followers(self, username):
        if self.user_min_follow == 0 and self.user_max_follow == 0:
            return True

        try:
            followers = self.get_followers_count(username)
            if followers < self.user_min_follow:
                self.logger.info(
                    f"Won't follow {username}: does not meet user_min_follow requirement"
                )
                return

            if self.user_max_follow != 0 and followers > self.user_max_follow:
                self.logger.info(
                    f"Won't follow {username}: does not meet user_max_follow requirement"
                )
                return

        except Exception as exc:
            self.logger.exception(exc)

        # Need "return True" here.

```

#### Config

```
max_like_for_one_tag: 5
follow_per_day: 50
follow_per_run: 50
follow_time: 360000
user_max_follow: 5000
user_min_follow: 10
tag_list: ['foo', 'bar', 'baz', ]
keywords: []
```

**Comment by marcularis04** [comment]
I also have the same problem

**Comment by wdstk** [comment]
I've issued pull request 😎

**Comment by manguszta** [comment]
Still doesn't work.

**Comment by nickgryg** [comment]
This issue has been already fixed since version 0.5.12.
